### PR TITLE
Fix the compilation of solo5-aarch64 and the TLS support

### PR DESCRIPTION
**Pull request by dinosaure**
/cc @palainp can you confirm that it's the right fix for `aarch64`?

### CODE REVIEW

**Comment by palainp** [comment]
Oh sorry I missed that thank you! I unfortunately can't test it due to lack of hardware :(

It looks good to me, perhaps you'd merge both liges into single one ?

**Comment by dinosaure** [comment]
> It looks good to me, perhaps you'd merge both liges into single one ?

I think it's fine. About testing, we still are under the experimental state about the `aarch64` support :+1:. Thanks for the quick review.

File: bindings/tls.c [hunk 1]
```diff
@@ -78,6 +78,7 @@ uintptr_t _solo5_tls_data_offset(uintptr_t tls)
 #if defined(__x86_64__) || defined(__powerpc64__)
     data = tls;
 #elif defined(__aarch64__)
+    data = tls;
     data = data + sizeof(struct tcb);
```
Comment (hannesm) [inline bindings/tls.c:82]: shouldn't these two lines be `data = tls + sizeof(struct tcb);`? The `data = data..` looks fishy.

Comment (dinosaure) [reply bindings/tls.c:82]: I will push on `master` directly with your change indeed :+1:.

**Pull request merged**